software/pangolin/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: pangolin
+description: Phylogenetic Assignment of Named Global Outbreak LINeages
+keywords:
+  - covid
+  - pangolin
+  - lineage
+tools:
+  - star:
+      description: |
+        Phylogenetic Assignment of Named Global Outbreak LINeages
+      homepage: https://github.com/cov-lineages/pangolin#pangolearn-description
+      manual: https://github.com/cov-lineages/pangolin#pangolearn-description
+params:
+  - outdir:
+      type: string
+      description: |
+        The pipeline's output directory. By default, the module will
+        output files into `$params.outdir/<SOFTWARE>`
+  - publish_dir_mode:
+      type: string
+      description: |
+        Value for the Nextflow `publishDir` mode parameter.
+        Available: symlink, rellink, link, copy, copyNoFollow, move.
+  - enable_conda:
+      type: boolean
+      description: |
+        Run the module with Conda using the software specified
+        via the `conda` directive
+  - singularity_pull_docker_container:
+      type: boolean
+      description: |
+        Instead of directly downloading Singularity images for use with Singularity,
+        force the workflow to pull and convert Docker containers instead.
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+  - assembly:
+      type: file
+      description: |
+        The virus genome assembly to be evaluated
```

Comment:
Does it have to be viral? 🤔

Reply:
Right now the tools seems very much covid-specific - but you're right, I can imagine this being used for other bugs.
So it can't hurt to revert this to "The genome assembly ..."

The people analysing the data from the next bacterial pandemic will appreciate it 😁

Reply:
Changed it :)